src/package/nuspec/Microsoft.NET.Test.Sdk.targets

```diff
@@ -18,7 +18,7 @@
      Generates Program file which contains the Main entry point
      ============================================================
   -->
-  <PropertyGroup>
+  <PropertyGroup Condition="'$(TargetFrameworkIdentifier)' == '.NETCoreApp'">
```

Comment:
A general thought (not specific to this PR):
For every build of a netcore test project, we will create this file. Does this imply that the test project will be compiled again always? (even if no actual code has changed)

If this is true, can it be avoided?